apps/api/src/app/integrations/usecases/create-novu-integrations/create-novu-integrations.usecase.ts

```diff
@@ -64,4 +66,9 @@ export class CreateNovuIntegrations {
       );
     }
   }
+
+  async execute(command: CreateNovuIntegrationsCommand): Promise<void> {
+    await this.createEmailIntegration(command);
+    await this.createSmsIntegration(command);
+  }
```

Comment:
before: when some of the SMS or Email credentials were not set it didn't create any integration, for example even if  Email credentials were set correctly

now: it does checks independently and will create Email/SMS integration if credentials are set